test/controllers/votes_controller_test.rb

```diff
@@ -0,0 +1,75 @@
+require "test_helper"
+
+describe VotesController do
+  describe "create" do
+    it "logged in user can vote" do
+      user = perform_login(users(:user_one))
+      work = works(:work_one)
+
+      # Arrange
+      vote_hash = {
+        vote: {
+          user_id: user.id,
+          work_id: work.id,
+        },
+      }
+
+      # Act-Assert
+      expect {
+        post work_votes_path(work.id), params: vote_hash
```

Comment:
If you notice there's no form in the upvote action.  Instead you just make a post request and the controller figures out the user by session and work by the route parameter.  

Also remember to test your flash notices.